examples/hello-world/ml-to-fl/tf/README.md

```diff
@@ -27,16 +27,14 @@ python3 ./code/cifar10_tf_original.py
 To transform the existing code to FL training code, we made the following changes:
 
 1. Import NVFlare Client API: ```import nvflare.client as flare```
-2. Import how to load/dump flatten weights: ```from nvflare.app_opt.tf.utils import get_flat_weights, load_flat_weights```
-3. Initialize NVFlare Client API: ```flare.init()```
-4. Receive aggregated/global FLModel from NVFlare side: ```input_model = flare.receive()```
-5. Load the received aggregated/global model weights into the model structure: ```load_flat_weights(net, input_model.params)```
-6. Evaluate on received aggregated/global model to get the metrics for model selection
-7. Construct the FLModel to be returned to the NVFlare side: ```output_model = flare.FLModel(params=get_flat_weights(net), xxx)```
-8. Send the model back to NVFlare: ```flare.send(output_model)```
+2. Initialize NVFlare Client API: ```flare.init()```
+3. Receive aggregated/global FLModel from NVFlare side: ```input_model = flare.receive()```
+4. Load the received aggregated/global model weights into the model structure: ```model.get_layer(k).set_weights(v)```
+5. Evaluate on received aggregated/global model to get the metrics for model selection
```

Comment:
Evaluate on the model with aggregated/global weights loaded: via regular .evaluate() function